### DISCUSSION TITLE
请教一下，simbot3.0可以监听@别人的消息嘛。

### DISCUSSION THREAD

**Discussion by QingShanYuanKe** [ForteScarlet/simpler-robot#440] [不要在这里发帖子了]
就比如 A：@B 。机器人能监听到被@人的nickname，即B。

**Comment by ForteScarlet** [comment]
`At` 属于**消息元素**，不属于事件。可参考 [标准消息](https://simbot.forte.love/docs/defition/message-overview/standard-message#at) 

或者参考源码类型

https://github.com/ForteScarlet/simpler-robot/blob/5b151d67635eb921741659ff4e5deccbae069cdc/simbot-apis/simbot-api/src/main/kotlin/love/forte/simbot/message/BaseStandardMessage.kt#L151-L201

如果有需要可通过代码逻辑进行获取并判断

**Reply by QingShanYuanKe** [reply]
我注意到atBot()是可以对@机器人本身进行监听的，而且在simbot2.0文档中存在anyAt方法的。simbot3.0现在暂时不支持对@别人的消息进行监听是嘛

**Reply by ForteScarlet** [reply]
你可以使用
```java
@Filter(targets = @Filter.Targets(atBot = true))
```

**Reply by QingShanYuanKe** [reply]
这个只能监听@机器人，@其他人就没办法监听到辽。

**Reply by ForteScarlet** [reply]
这个目前的确没有提供默认参数，你可以通过代码逻辑来实现匹配。如果有需求可以通过 [issues](https://github.com/ForteScarlet/simpler-robot/issues) 或者 [pr](https://github.com/ForteScarlet/simpler-robot/pulls) 进行反馈或协助。

**Reply by QingShanYuanKe** [reply]
好的 感谢解惑！
